### PR TITLE
Bootstrap - Meetings page

### PR DESCRIPTION
**Pull request by KimberleyCook**
Update the Meeting page to use Bootstrap 5.

I've done a small design update on this page using the Bootstrap `badge` component. Designs screenshots are below:

| Event already happened before | Event already happened after |
|-------|------|
|Screenshot 2020-11-20 at 16 23 34|Screenshot 2020-11-20 at 16 18 15|

| Suspended account before | Suspended account after
|-------|------|
|Screenshot 2020-11-20 at 16 24 53|Screenshot 2020-11-20 at 16 18 34|

| Event full before | Event full after
|-------|------|
|Screenshot 2020-11-20 at 16 26 30|Screenshot 2020-11-20 at 16 18 54|

### CODE REVIEW

File: app/views/meetings/_meeting_actions.html.haml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 - if @meeting.venue.present?
   %h4 Venue
-  %p
+  %p.mb-1
     %strong= @meeting.venue.name
-  .monthly-host-logo
+  =link_to @meeting.venue.website, class: 'border-0' do
```
Comment (KimberleyCook) [inline app/views/meetings/_meeting_actions.html.haml:5]: I also made the company logo link to their website.

File: spec/features/viewing_a_meeting_spec.rb [hunk 2]
```diff
@@ -25,11 +25,11 @@
       visit meeting_path(meeting)
       login_mock_omniauth(member, 'Log in')
 
-      click_on 'Attend'
+      click_on 'RSVP here'
 
       expect(page).to have_content('Your RSVP was successful. We look forward to seeing you at the Monthly!')
 
-      click_on "Can't make it anymore? Click here to cancel your spot."
```
Comment (matyikriszta) [inline spec/features/viewing_a_meeting_spec.rb:32]: @KimberleyCook any reason why we removed the period from this sentence? I think I prefer it with period.

Comment (KimberleyCook) [reply spec/features/viewing_a_meeting_spec.rb:32]: Yeah I removed them as I think they looked odd 😂

**Review by matyikriszta — APPROVED** [review]
👍 looks good, love the badges

**Pull request merged**